### ISSUE TITLE
Python GUI for consistency (Qt)

### ISSUE THREAD

**Issue by aitor-gomila** [symphonly/figaro#50]
**Is your feature request related to a problem? Please describe.**
It's not normal that the lib is written in Python and the GUI in electron (nodejs). This may cause many security problems.

**Describe the solution you'd like**
A new GUI written with Python PySide6 or some other gui framework.

**Describe alternatives you've considered**
You're also running the production app in dev. this is a problem!

**Additional context**

**Comment by MattMoony** [comment]
Hi!

First of all, thanks a lot for your interest in this project of mine - it's really cool to see people interested in the stuff I've created!

As for the issue you have raised: Yes, I'm aware that combining Python and Electron is not the usual way of developing desktop apps, however, that's due to the fact that I didn't really plan on developing a GUI for Figaro in the first place and I then just looked for a quick and easy extension to make the project more user-friendly - in my case, this was Electron, as I already had some experience with it.

**TLDR**: thx for the feedback, will definitely come back to it when I have more time on my hands than I do now :)

Cheers, Matthias

**Comment by Tilix4** [comment]
I believe [DearPyGui](https://github.com/hoffstadt/DearPyGui) would be lighter and the imGUI approach is easier to handle. (I strongly hate QT ;) )

**Comment by ghost** [comment]
Agreed

**Comment by boboyiyi** [comment]
您发给我的信件已经收到。谢谢!
